src/main/java/org/primefaces/component/panelgrid/PanelGridRenderer.java

```diff
@@ -39,15 +39,19 @@
 
 public class PanelGridRenderer extends CoreRenderer {
 
+    public static final String LAYOUT_TABULAR = "tabular";
+    public static final String LAYOUT_GRID = "grid";
+    public static final String LAYOUT_FLEX = "flex";
+
     @Override
     public void encodeEnd(FacesContext context, UIComponent component) throws IOException {
         PanelGrid grid = (PanelGrid) component;
 
-        if (grid.getLayout().equals("tabular")) {
+        if (grid.getLayout().equals(LAYOUT_TABULAR)) {
             encodeTableLayout(context, grid);
```

Comment:
Reverse like you have below LAYOUT_TABLE.equals(grid.getLayout()).  Also should we do equalsIgnoreCase() everywhere on these so if someone accidentally puts "Flex" instead of "flex they don't get dinged?